Dockerfile

```diff
@@ -83,7 +83,7 @@ WORKDIR /home/argocd
 ####################################################################################################
 # Argo CD UI stage
 ####################################################################################################
-FROM --platform=$BUILDPLATFORM docker.io/library/node:22.9.0@sha256:69e667a79aa41ec0db50bc452a60e705ca16f35285eaf037ebe627a65a5cdf52 AS argocd-ui
+FROM --platform=$BUILDPLATFORM docker.io/library/node:22.9.0@sha256:f2f00f9d041d78b2a76ce80894602a5260ccada74ac294eeb07925a51707d564 AS argocd-ui
```

Comment:
Weird that it didn't update the version number...

Reply:
I can see it is updated now, but here conflicts, maybe worth to close this PR so it will reopen